src/dotnet/commands/dotnet-new/CSharp_Web/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM microsoft/aspnet:1.0.0-rc1-update1
+
+RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
+RUN apt-get -qq update && apt-get install -qqy sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
+
+COPY . /app
+WORKDIR /app
+RUN ["dnu", "restore"]
+
+EXPOSE 5000/tcp
+ENTRYPOINT ["dnx", "-p", "project.json", "Microsoft.AspNet.Server.Kestrel", "--server.urls", "http://0.0.0.0:5000"]
```

Comment:
Yea this looks old. I copied it from the output of yeoman generator. I am not sure if this file is even needed. @sayedihashimi, can this file be removed?